packages/dashboard/next.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  async redirects() {
+    const redirects = [];
+
+    if (process.env.ACCOUNTS_MAINTENANCE === "true") {
+      redirects.push({
+        source: "/((?!maintenance$).*)",
```

Comment:
`nit` could've dropped a line here explaining the neg. lookahead

Reply:
I'll remove this once we're done with migration, let's just leave it as is for now